### PR TITLE
Smaller code cleanups

### PR DESCRIPTION
**Pull request by sabbelasichon**

### CODE REVIEW

File: src/Command/SelfUpdateCommand.php [hunk 1]
```diff
@@ -4,22 +4,22 @@
 
 namespace TYPO3\Surf\Command;
 
+use Humbug\SelfUpdate\Strategy\GithubStrategy;
```
Comment (simonschaufi) [inline src/Command/SelfUpdateCommand.php:7]: what happened here? :thinking:

File: src/Application/BaseApplication.php [hunk 2]
```diff
@@ -115,7 +115,7 @@ public function registerTasks(Workflow $workflow, Deployment $deployment): void
             ->addTask(SymlinkReleaseTask::class, SimpleWorkflowStage::STEP_09_SWITCH, $this)
             ->addTask(CleanupReleasesTask::class, SimpleWorkflowStage::STEP_10_CLEANUP, $this);
 
-        if ($this->hasOption('lockDeployment') && $this->getOption('lockDeployment') === true) {
+        if ($this->provideBoolOption('lockDeployment')) {
```
Comment (simonschaufi) [inline src/Application/BaseApplication.php:118]: does this make sure it is also true and not false?

Comment (sabbelasichon) [reply src/Application/BaseApplication.php:118]: Yes. Sure. It provides the value as a boolean type. Could be either false or true.

File: src/Command/SelfUpdateCommand.php (hunk 1 above)
Comment (sabbelasichon) [reply src/Command/SelfUpdateCommand.php:7]: Oh, this is rector mess up.

File: src/Application/BaseApplication.php (hunk 2 above)
Comment (simonschaufi) [reply src/Application/BaseApplication.php:118]: Yes but in this case the value needs to be true which is not checked any more, or?

File: src/Command/SelfUpdateCommand.php (hunk 1 above)
Comment (simonschaufi) [reply src/Command/SelfUpdateCommand.php:7]: can you please clean up that mess?

File: src/Application/BaseApplication.php (hunk 2 above)
Comment (sabbelasichon) [reply src/Application/BaseApplication.php:118]: $this->provideBoolOption('lockDeployment') means true, right?

**Pull request merged**